convert2rhel/systeminfo.py

```diff
@@ -33,11 +35,26 @@
 from convert2rhel.toolopts import tool_opts
 
 
+# Supported conversion path to RHEL
+RELEASE_VER_MAPPING = {
+    "CentOS Linux": {
```

Comment:
I think it is better to have them separated - as it is another OS. Changes to OL comes more frequently. Even though now we have identical lines

Reply:
Until CentOS or Oracle starts to differ there isn't going to be any differences that would cause problems. If we add a mapping for 8.5 now, it will start working for Oracle when their minor version is released and it would introduce no issues for CentOS. There would only be a problem if Oracle added a new minor version that is not based off of RHEL. Does that make sense?

Reply:
ok, just to double confirm. My position is based on just a design sense. No strong arguments